### PR TITLE
DR-2291 Add a new front end configuration chart

### PR DESCRIPTION
**Pull request by nmalfroy**
This allows us to create a k8s resource for configuring an SSL policy for things like prometheus and grafana using an annotation on the ingress config like:
`networking.gke.io/v1beta1.FrontendConfig: "<what we specify in the sslPolicy value>"`

Note: the default ssl policy is set to one that we create in Terraform that enforces that TLS 1.2 is used by the client

Note: this also does http -> https redirects

### CODE REVIEW

**Review by samanehsan — APPROVED** [review]

File: charts/frontend-config/templates/frontendconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+kind: FrontendConfig
+metadata:
+  name: {{ template "frontend-config.fullname" . }}
+  labels:
+    {{- include "frontend-config.labels" . | nindent 4 }}
+spec:
+  sslPolicy: {{ .Values.sslPolicy }}
+  redirectToHttps:
+    enabled: true
+    responseCodeName: MOVED_PERMANENTLY_DEFAULT
```
Comment (tlangs) [inline charts/frontend-config/templates/frontendconfig.yaml:12]: do we want a newline?

File: charts/frontend-config/values.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+---
+sslPolicy: global-ssl-policy
```
Comment (tlangs) [inline charts/frontend-config/values.yaml:2]: newline here too?

**Review by tlangs — APPROVED** [review]
Approved, but with extremely minor comments.

File: charts/frontend-config/.helmignore [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+.git/
+.gitignore
+.bzr/
+.bzrignore
+.hg/
+.hgignore
+.svn/
+# Common backup files
+*.swp
+*.bak
```
Comment (pshapiro4broad) [inline charts/frontend-config/.helmignore:15]: `.bak`, really?

**Review by pshapiro4broad — APPROVED** [review]

File: charts/frontend-config/templates/frontendconfig.yaml (hunk 1 above)
Comment (nmalfroy) [reply charts/frontend-config/templates/frontendconfig.yaml:12]: ah right

File: charts/frontend-config/.helmignore (hunk 3 above)
Comment (nmalfroy) [reply charts/frontend-config/.helmignore:15]: lol...yeah...some copy pasta

**Pull request merged**